README.md

```diff
@@ -14,50 +14,59 @@ Requirements
 -	[Terraform](https://www.terraform.io/downloads.html) 0.13.x
 -	[Go](https://golang.org/doc/install) 1.14 (to build the provider plugin)
 
+Upgrade state
+-------
+To switch state from the deprecated provider (builds <= 0.3.64) please run
+```sh
+terraform state replace-provider registry.terraform.io/g-core/gcorelabs registry.terraform.io/g-core/gcore
+```
+
 Building the provider
 ---------------------
 ```sh
-$ mkdir -p $GOPATH/src/github.com/terraform-providers
-$ cd $GOPATH/src/github.com/terraform-providers
-$ git clone https://github.com/G-Core/terraform-provider-gcore.git
-$ cd $GOPATH/src/github.com/terraform-providers/terraform-provider-gcore
-$ make build
+GOPATH=$(go env GOPATH)
+mkdir -p $GOPATH/src/github.com/terraform-providers
+cd $GOPATH/src/github.com/terraform-providers
+git clone https://github.com/G-Core/terraform-provider-gcore.git
+cd $GOPATH/src/github.com/terraform-providers/terraform-provider-gcore
+make build
 ```
 
 ### Override Terraform provider
 
-To override terraform provider for development goals you do next steps: 
+To override terraform provider for development goals you do next steps:
 
-create Terraform configuration file
+- create Terraform configuration file and point provider to development path
+- comment out the override if you want to use published binary from the upstream
 ```shell
-$ touch ~/.terraformrc
-```
+cat > ~/.terraformrc << EOF
+provider_installation {
 
-point provider to development path
-```shell
-provider_installation { 
- 
-  dev_overrides { 
-      "local.gcore.com/repo/gcore" = "/<dev-path>/terraform-provider-gcore/bin" 
-  } 
- 
-  # For all other providers, install them directly from their origin provider 
-  # registries as normal. If you omit this, Terraform will _only_ use 
-  # the dev_overrides block, and so no other providers will be available. 
-  direct {} 
+  dev_overrides {
+      "local.gcore.com/repo/gcore" = "$(go env GOPATH)/src/github.com/terraform-providers/terraform-provider-gcore/bin"
+      "registry.terraform.io/g-core/gcore" = "$(go env GOPATH)/src/github.com/terraform-providers/terraform-provider-gcore/bin"
+  }
+
+  # For all other providers, install them directly from their origin provider
+  # registries as normal. If you omit this, Terraform will _only_ use
+  # the dev_overrides block, and so no other providers will be available.
+  direct {}
 }
+EOF
 ```
 
-add `local.gcore.com/repo/gcore` to .tf configuration file
+Optionally specify `local.gcore.com/repo/gcore` in main.tf configuration file
 ```shell
 terraform {
-  required_version = ">= 0.13.0"
-
   required_providers {
     gcore = {
-      source = "local.gcore.com/repo/gcore"
+      source  = "G-Core/gcore"
+      version = ">= 0.3.65"
+      # source = "local.gcore.com/repo/gcore"
```

Comment:
remove commented out lines

Reply:
why ? this is an illustration for the comment 
 58 Optionally specify `local.gcore.com/repo/gcore` in main.tf configuration file